src/test/java/com/wikia/webdriver/pageobjectsfactory/pageobject/adsbase/AdsFandomObject.java

```diff
@@ -9,6 +9,9 @@
 import org.openqa.selenium.WebElement;
 
 public class AdsFandomObject extends AdsBaseObject {
+  private static final By F2_MOBILE_NAVIGATION_BAR = By.cssSelector(".global-navigation-mobile");
+  private static final By F2_DESKTOP_NAVIGATION_BAR = By.cssSelector(".wds-global-navigation");
```

Comment:
I think as far as those consts are in `AdsFandomObject` we don't need to add F2 prefix to it